qutip/qobj.py

```diff
@@ -1408,11 +1412,11 @@ def tidyup(self, atol=settings.auto_tidyup_atol):
         ----------
         atol : float
             Absolute tolerance used by tidyup. Default is set
-            via qutip global settings parameters.
+            via qutip global settings parameters in :obj:`.Options`.
```

Comment:
The link here isn't correct.  The linked options are the solver options, whereas the global atol settings are in `qutip.settings`.